soc/arm/silabs_exx32/efm32pg12b/dts_fixup.h

```diff
@@ -26,4 +26,7 @@
 #define DT_GPIO_GECKO_PORTE_NAME	DT_SILABS_EFM32_GPIO_PORT_4000A0C0_LABEL
 #define DT_GPIO_GECKO_PORTF_NAME	DT_SILABS_EFM32_GPIO_PORT_4000A0F0_LABEL
 
+#define DT_WDT_0_NAME				DT_INST_0_SILABS_GECKO_WDOG_LABEL
```

Comment:
Remove these defines, we should utilize aliases instead.

Reply:
Removed